e2e-tests/version-service/run

```diff
@@ -61,11 +60,31 @@ function check_telemetry_transfer() {
 		| jq 'del(."grpc.time_ms")' \
 			>${tmp_dir}/${telemetry_state}_telemetry.version-service.log.json
 
-	diff ${test_dir}/compare/${telemetry_state}_telemetry.version-service.log.json <(grep -f ${tmp_dir}/${telemetry_state}_telemetry.version-service.log.json ${test_dir}/compare/${telemetry_state}_telemetry.version-service.log.json)
+	local telemetry_log_file="${telemetry_state}_telemetry.version-service.log${OPERATOR_NS:+-cw}.json"
+	# telemetry was disabled in CR but in operator not
+	if [ ${cr_vs_channel} == 'disabled' -a "${telemetry_state}" == 'enabled' ]; then
+		# operator fallback VS should have telemetry
+		diff ${test_dir}/compare/${telemetry_log_file} <(grep -f ${tmp_dir}/${telemetry_state}_telemetry.version-service.log.json ${test_dir}/compare/${telemetry_log_file})
+		# CR VS should not have telemetry
+		[[ -s "${tmp_dir}/enabled_telemetry.version-service-cr.log.json" ]] && exit 1
+	fi
+
+	local telemetry_cr_log_file="${telemetry_state}_telemetry.version-service-cr.log${OPERATOR_NS:+-cw}.json"
+	# telemetry was disabled in operator but not in CR
+	if [ ${cr_vs_channel} == 'recomended' -a "${telemetry_state}" == 'enabled' ]; then
+		# cr VS should have telemetry
```

Comment:
`recommended`
 Why we check this case? We don't run check_telemetry_transfer with this combination.

Reply:
fixed